js/privatebin.js

```diff
@@ -3525,6 +3525,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             $password,
             $passwordInput,
             $rawTextButton,
+	    $downloadTextButton,
```

Comment:
Tabulation

Reply:
Hint: It's tabs, but should be spaces. You seem to miss some editorconfig plugin or so here… :wink:

Reply:
fixed in 853a4f386fa9a870cb2aa3616d43b61d0f10ec5f